src/ekat/kokkos/ekat_subview_utils.hpp

```diff
@@ -316,6 +315,177 @@ subview_1(const ViewLR<ST******,Props...>& v,
       v.impl_track(),vm);
 }
 
+// ================ Multi-sliced Subviews ======================= //
+// e.g., instead of a single-entry slice like v(:, 42, :), we slice over a range
+// of values, as in v(:, 27:42, :)
+// this means that the subview has the same rank as the "parent" view
```

Comment:
I meant to refer to the view from which the subview is being taken, but I agree it's best to avoid ambiguity. Will go with "source view" unless there's objections